Client/Frontend/Widgets/ThumbnailCell.swift

```diff
@@ -8,11 +8,16 @@ private let BorderColor = UIColor(rgb: 0xeeeeee)
 private let LabelFont = UIFont(name: "FiraSans-Regular", size: 11)
 private let LabelColor = UIAccessibilityDarkerSystemColorsEnabled() ? UIColor.blackColor() : UIColor.darkGrayColor()
 private let CellInsets = UIEdgeInsetsMake(8, 8, 8, 8)
-private let TextMargin = 5
+
+private let PlaceholderImage = UIImage(named: "defaultFavicon")
+
+struct ThumbnailCellUX {
+    static let ImageAspectRatio: Float = 1.5
```

Comment:
Yeah, it is confusing. The aspect ratio here refers to the image; the aspect ratio on TopSitesPanel refers to the actual cell. We need to explicitly specify the former so that we take thumbnails that are the correct size, and I think we need to explicitly specify the latter so the collection view size can be calculated. You implemented the collection view sizing stuff, so happy to hear suggestions for improving this :)